components/SpotlightShare/SpotlightShare.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent } from 'react';
+import Modal from 'react-modal';
+import { Button } from '../Button';
+import classNames from 'classnames';
+import { FaFacebookF, FaTwitter } from 'react-icons/fa';
```

Comment:
We already have SVG for to use for social icons. Please use those. 
They're in the footer ... https://development-initiatives.surge.sh/page-templates/22-spotlight#

![image](https://user-images.githubusercontent.com/5672438/76592308-04060780-6504-11ea-9144-21d8fec903a6.png)

Reply:
If there's any you lack, notify Ben.

Reply:
I suggest you create a component that abstracts this ... probably takes in a name and renders the icon ... responds to clicks e.t.c.

Reply:
Alright